### ISSUE TITLE
Sending Post request

### ISSUE THREAD

**Issue by Nedervino** [nfarina/homebridge#479]
I have just loaded the new version of homebridge on my mac, but the old accessory js file I had is not compatible anymore. How would I be able to send a post request with a body using the new plugin system (possibly by altering one of the existing plugin files)? Thanks in advance

**Comment by damianromanowski** [comment]
@Nedervino Check out https://github.com/rudders/homebridge-http

It does HTTP POSTs

**Comment by scottbots** [comment]
@damianromanowski Hi there, I was trying to use the library you linked and I have no problem at all getting the http GET command to work but I'm having trouble with the HTTP POSTs.  The library example config.json shows an HTTP GET example but not a POST.  Any help to pointing me in the right direction would be appreciated.  Below is my working HTTP GET example.  I'm not quite sure how to modify this to work instead with the POST request instead so I can also send my payload/message body.  When I attempted changing the http_method to "POST" and modifying the code, I had no luck.  Maybe I was not formatting it correctly?  

The reason why I need to use POST (and not GET) is because I'm connecting to a particle photon micro controller and it must receive the request as a POST command to trigger any functions I have created.  The IFTTT maker channel was a quick work around for me and it also confirmed that the GET requests were working fine for me.  Any help would be appreciated and if any other info is needed I can provide it.    

`
{

"accessory": "Http",
"name": "light",
"switchHandling": "yes",
"http_method": "GET",
"on_url": "https://maker.ifttt.com/trigger/on/with/key/xxxxxxxxxxxx",
"off_url": "https://maker.ifttt.com/trigger/off/with/key/xxxxxxxxxxxx",
"status_url":  "",
"service": "Light",
"brightnessHandling": "no",
"brightness_url":     "",
"brightnesslvl_url":  "",
"username" : "",
"password" : "",
"sendimmediately": ""
},
`
